packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -296,7 +298,7 @@ exit 0
 %doc LICENSE
 %dir %{_localstatedir}/cache/katello-agent/
 
-%if %{dnf_install} || %{yum_install}
+%if %{dnf_install} || %{yum_install} || %{zypper_install}
```

Comment:
Is there a build that does not trigger one of these at this point?

Reply:
same as above